src/Model/Button/AccountLink.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace BokaiTid\FBMessenger\Model\Button;
+
+use Tgallice\FBMessenger\Model\Button;
+
+class AccountLink extends Button
+{
+    /**
+     * @var string $url
+     */
+    private $url;
+
+    /**
+     * @param string $url
+     */
+    public function __construct($url)
+    {
+        parent::__construct(Button::TYPE_ACCOUNT_LINK);
+
+        $this->url = $url;
+    }
+
+    /**
+     * @return string
+     */
+    public function getUrl()
+    {
+        return $this->url;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function jsonSerialize()
+    {
+        $json = parent::jsonSerialize();
+        $json['url'] = $this->url;
+
+        return $json;
+    }
+}
```

Comment:
Sure, i'm just not sure how to edit the file yet without creating a new commit. (haha shot me xD)